add document: how to make a patch for export-diff cmd (#117)
cybozu-go/mantle: ceph/README.md

## Procedure to update
### Update ceph version

> **In the `cybozu/ceph` repository,** imports all of the contents from upstream git, including tags.

Reviewer: ```suggestion
**In the `cybozu/ceph` repository,** sync with the contents from upstream git (`ceph/ceph`),  including tags.
```

```

### Update the patch

> **In the `cybozu/ceph` repository,** the export-diff patch is created by each target Ceph version. A branch is created for each Ceph version. If there is no branch for the target version, You can create it like the following.

Reviewer: ```suggestion
**In the `cybozu/ceph` repository,** you checkout the target branch (`rbd-export-vx.y.z`). If it doesn't exist yet, it's necessary to create it and apply the newest patch as follows.
```

```sh

> # at working directory for https://github.com/ceph/ceph.git

Reviewer: ```suggestion
# at working directory for https://github.com/cybozu/ceph.git
```

VERSION=16.2.4

> git checkout -b rbd-export-v$VERSION tags/v$VERSION

Reviewer: ```suggestion
git checkout -b rbd-export-v$VERSION tags/v$VERSION
git apply ceph/export-diff.patch
```

```

> You commit & push to the branch in the same way as normal development, but **you do not create an RP, and do not merge to the main branch.** You simply save the changes to the branch.

Reviewer: ```suggestion
You modify code, commit the changes, and push to the branch in the same way as normal development. **You must not create a PR corresponding to this branch**.
```

```sh
git diff tags/v$VERSION > export-diff.patch
```

> The patches are stored in the `ceph` directory in the `cybozu-go/mantle` repository. Please copy them manually.

Reviewer: ```suggestion
You store the up-to-date export-diff.patch as [`ceph/export-diff.patch`](https://github.com/cybozu-go/mantle/blob/main/ceph/export-diff.patch).
```
